src/screens/Messages/List/ChatListItem.tsx

```diff
@@ -203,6 +204,21 @@ function ChatListItemReady({
       onFocus={onFocus}
       onBlur={onMouseLeave}
       style={[a.relative]}>
+      <Link
+        to={`/messages/${convo.id}`}
+        label={displayName}
+        style={[
+          a.z_10,
+          a.absolute,
+          {top: tokens.space.md, left: tokens.space.lg},
+        ]}>
```

Comment:
Think we can remove this (and looks like it's going to the wrong place anyway haha)

Reply:
lmao whoops I clicked it and it went to the right place so I never gave it a second thought 😭